var/spack/repos/builtin/packages/py-jupytext/package.py

```diff
@@ -16,6 +16,7 @@ class PyJupytext(PythonPackage):
 
     maintainers = ['vvolkl']
 
+    version('1.13.6', sha256='c6c25918ddb6403d0d8504e08d35f6efc447baf0dbeb6a28b73adf39e866a0c4')
     version('1.13.0', sha256='fb220af65d2bd32d01c779b0e935c4c2b71e3f5f2f01bf1bab10d5f23fe121d4')
 
```

Comment:
Missing the following deps:
```
    python_requires="~=3.6",                                                             
```
```
requires = ["jupyter_packaging~=0.7.9", "jupyterlab>=3.0.0,==3.*", "setuptools>=40.8.0", "wheel"]
```

Reply:
Done, save for the jupyter ones, which are only needed to use jupytext as  a jupyter extension. I  cannot test this at the moment, so I'm adding as a todo.

Reply:
Is there any reason not to add them anyway? I'm not sure what pip does if the required deps are missing when it tries to build.

Reply:
jupytext builds fine without them (meaning the jupytext executable). It'll build fine with them also, so I'm happy to add them, I just won't debug the jupyter extension right now.